saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py

```diff
@@ -56,6 +56,204 @@ def test_category_deleted(category, subscription_category_deleted_webhook):
+):
+    # given
+    webhooks = [subscription_shipping_price_created_webhook]
+    event_type = WebhookEventAsyncType.SHIPPING_PRICE_CREATED
+
+    shipping_method_id = graphene.Node.to_global_id(
+        "ShippingMethodType", shipping_method.id
+    )
+    shipping_zone_id = graphene.Node.to_global_id(
+        "ShippingZone", shipping_method.shipping_zone.id
+    )
+
+    # when
+    deliveries = create_deliveries_for_subscriptions(
+        event_type, shipping_method, webhooks
+    )
+
+    # then
+    expected_payload = json.dumps(
+        {
+            "shippingMethod": {
+                "id": shipping_method_id,
+                "name": shipping_method.name,
+                "channelListings": [
+                    {"channel": {"name": sl.channel.name}}
+                    for sl in shipping_method.channel_listings.all()
+                ],
+            },
+            "shippingZone": {
+                "id": shipping_zone_id,
+                "name": shipping_method.shipping_zone.name,
+            },
+            "meta": None,
+        }
+    )
+
+    assert deliveries[0].payload.payload == expected_payload
+    assert len(deliveries) == len(webhooks)
+    assert deliveries[0].webhook == webhooks[0]
+
+
+def test_shipping_price_updated(
+    shipping_method, subscription_shipping_price_updated_webhook
+):
+    # given
+    webhooks = [subscription_shipping_price_updated_webhook]
+    event_type = WebhookEventAsyncType.SHIPPING_PRICE_UPDATED
+    shipping_method_id = graphene.Node.to_global_id(
+        "ShippingMethodType", shipping_method.id
+    )
+    shipping_zone_id = graphene.Node.to_global_id(
+        "ShippingZone", shipping_method.shipping_zone.id
+    )
+
+    # when
+    deliveries = create_deliveries_for_subscriptions(
+        event_type, shipping_method, webhooks
+    )
+
+    # then
+    expected_payload = json.dumps(
+        {
+            "shippingMethod": {
+                "id": shipping_method_id,
+                "name": shipping_method.name,
+                "channelListings": [
+                    {"channel": {"name": sl.channel.name}}
+                    for sl in shipping_method.channel_listings.all()
+                ],
+            },
+            "shippingZone": {
+                "id": shipping_zone_id,
+                "name": shipping_method.shipping_zone.name,
+            },
+            "meta": None,
+        }
+    )
+
+    assert deliveries[0].payload.payload == expected_payload
+    assert len(deliveries) == len(webhooks)
+    assert deliveries[0].webhook == webhooks[0]
+
+
+def test_shipping_price_deleted(
+    shipping_method, subscription_shipping_price_deleted_webhook
+):
+    # given
+    webhooks = [subscription_shipping_price_deleted_webhook]
+    event_type = WebhookEventAsyncType.SHIPPING_PRICE_DELETED
+    shipping_method_id = graphene.Node.to_global_id(
+        "ShippingMethodType", shipping_method.id
+    )
+
+    # when
+    deliveries = create_deliveries_for_subscriptions(
```

Comment:
Keep in mind that you will get here an object that was deleted (so ID will be missing). See mutation for product_deleted, we handled there this situation